clouddriver-titus/clouddriver-titus.gradle

```diff
@@ -1,5 +1,6 @@
 dependencies {
   compile project(":clouddriver-core")
+  compile project(":clouddriver-aws")
```

Comment:
this is unfortunately necessary for now - there is a HealthProvidingCachingAgent interface in clouddriver-aws referenced by clouddriver-titus